Source/WebKit/UIProcess/Extensions/Cocoa/API/WebExtensionContextAPIActionCocoa.mm

```diff
@@ -127,15 +127,15 @@
 
     id parsedIcons = parseJSON(iconsJSON, JSONOptions::FragmentsAllowed);
     if (auto *dictionary = dynamic_objc_cast<NSDictionary>(parsedIcons))
-        action.value()->setIcons(dictionary);
+        Ref { action.value() }->setIcons(dictionary);
```

Comment:
You should just pull `action.value()` out into a local `Ref` above, so this code is not so ugly and repetitive.

Reply:
That's a good point--thank you! Fixed.